README.md

```diff
@@ -1,4 +1,85 @@
-# devtools
+# Devtools
 
-Collection of scripts (for now) to help developers.
+Collection of scripts and additional information to help developers.
 
+## Manage development installation
+
+To clone/install/update all ubermag subpackages we have a convenience script
+`manage-ubermag.py`. To get a list of all possible options run:
+
+```bash
+python manage-ubermag.py -h
+```
+
+We assume/create a directory structure where all subpackages are contained
+within one base directory, here called `ubermag`:
+
+    ubermag
+      +-- discretisedfield
+      |     +-- .github
+      |     +-- ...
+      |
+      +-- micromagneticdata
+      |     +-- .github
+      |     +-- ...
+      |
+      +-- setup-ubermag.py
+
+### Clone and install packages
+
+To get started create the base directory, copy `setup-ubermag.py` to that
+directory, and change into that directory. Your directory layout should then be:
+
+    ubermag
+      +-- setup-ubermag.py
+
+#### 1. Create and activate conda environment
+
+Create a new environment, here called `ubermagdev`, and install the most basic
+packages using `conda` (everything else will be installed via `pip`).
+
+```bash
+conda create -n ubermagdev -c conda-forge python=3.8 pip oommf
```

Comment:
I've used `3.8` because this is the minimal version that we currently support (so no additional features are available locally during development). I guess we don't want to already increase the minimal required Python version. Any thoughts?

Reply:
That's a good reason for going with 3.8. (I hadn't thought about accidental use of language features in newer versions.)